src/main/java/com/danit/controllers/employee/GroupTrainingController.java

```diff
@@ -0,0 +1,72 @@
+package com.danit.controllers.employee;
+
+import com.danit.exceptions.EntityNotFoundException;
+import com.danit.models.eployee.GroupTraining;
+import com.danit.repositories.employee.GroupTrainingRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class GroupTrainingController {
+  @Autowired
+  private GroupTrainingRepository groupTrainingRepository;
+
+  @GetMapping("/grouptraining")
```

Comment:
We usually separate words within endpoint names with underscore. So I suggest to use '/group_training' here.

Reply:
ok